structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,135 @@
+
+Each community gets a single vote. They are free to write their own process for determining their vote, but it must follow those rules:
+
+- This process must be democratic and give a voice to any community member who is able to join the Community Collab server under the Collaborator role.
+- Each eligible voter must have single, equal voting power.
+- The process can remove the voting power under some reasonable inactivity threshold.
+
+Once a community has decided their vote, one member can report their vote in the thread, mentioning the community name, and the option being voted for, with corresponding proof (such as a screenshot of an internal poll).
+
+Each community also has the option of skipping its vote if none of its members has an opinion on the poll. Such instances should be reported the same way.
+
+Polls must include a closing date of at least ten days. If required, the closing date can be pushed away by a maximum of a month, upon the informal agreement with the majority of the other communities. During that period, any Collaborator can use their veto right (see below), if allowed by the process.
+
+Once the poll is closed, the vote can be interpreted, depending on the specific process this vote is part of, if any.
+
+## Veto rights
+
+While the vote is active, any Collaborator or Manager can use their veto right by sending a message inside the thread, detailing their reasoning.
+
+If another Collaborator or Manager believes the reasoning isn’t satisfying, they can start the overruling process. If so, another poll is set up, if more than half of the individual votes are positive after three days, the veto is discarded.
+
+If no Collaborator or Manager overrules the veto after three days, the vote is considered as failed right away. The Managers are free to only communicate the result to the Invitees at a later date to keep the veto secret.
+
+## Process: adding a community
+
+Any community, ideally having their main user space on Discord, centred around Minecraft, can apply to join the Community Collab program. The process is defined as follows:
+
+1. A member of the community holding administrative or moderation powers needs to get in touch with a Manager to start the process. The following questions should be asked to the community and collected, along with an invite to the server:
+    - What is the main focus of the server?
+    - Why is your community looking to join the Community Collab program?
+    - What is your current ruleset?
+    - Who is currently part of your staff team, and who would have access to Community Collab?
+2. Once those answers are collected, the Manager can inform all the Collaborators about the community, and start a discussion to gauge the interest of the community.
+3. The members of the community holding either administrative or moderation powers are invited to the Community Collab server under the Invitee role and a channel is set up to allow discussion between Collaborators and Invitee.
```

Comment:
Why? Also, if the invitee idea only exists for this purpose, it should probably be folded into the process; though I don't see a reason to bring in more than one or two representatives from the community that's applying.

Reply:
I don't see a reason to limit the amount of persons able to join as invitees. I'd rather meet all the people who are able to join the server later now, than discover them later. Also the invitee role can be used during the removal process, and may be used by future processes, such as individual collaborators.

Reply:
Hmm, okay, I see